wordpress-mobile/WordPress-iOS
# Merge Today Widget feature branch into develop

It's a very big PR, containing all the previously merged PRs of the [project](https://github.com/wordpress-mobile/WordPress-iOS/projects/18)

Fixes #NA

To test:

- build/run and make sure you can install the today widget and it works as expected.

PR submission checklist:
